pkgs/by-name/sh/shittier/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "shittier";
+  # No tagged release on GitHub yet
+  # Commit corresponds to release tagged as 0.1.1 on [npm](https://www.npmjs.com/package/shittier)
+  # See issue https://github.com/rohitdhas/shittier/issues/7
+  version = "unstable-2023-12-22";
```

Comment:
```suggestion
  version = "0-unstable-2023-12-22";
```

Reply:
Done. But when did the versioning syntax change for unstable versions and why?